src/main.ts

```diff
@@ -100,7 +101,9 @@ async function setPermissions() {
 }
 
 async function waitForInternetConnection() {
-    while (!net.isOnline()) {
+    const check = await fetch(`${cloudHost}/ping`);
```

Comment:
This is VERY bad because if the server goes offline, no one will be able to open GoofCord. And if the check is not successful the first time, it will be stuck in an infinite loop because the request is not sent again

Reply:
Alr